extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/model/ApiGatewayAuthorizerContext.java

```diff
@@ -42,6 +42,11 @@
     // Methods - Public
     //-------------------------------------------------------------
 
+    @JsonAnyGetter
+    public Map<String, String> getContextProperties() {
+        return contextProperties;
+    }
+
     @JsonAnyGetter
     public String getContextValue(String key) {
```

Comment:
public String getContextValue(String key).  The @JsonAnyGetter annotation should be removed from this method

Reply:
Removed